render/tests/test-attributes.js

```diff
@@ -625,99 +627,32 @@ o.spec("attributes", function() {
 			var c = makeSelect(1)
 			var d = makeSelect("2")
 
-			render(root, [a])
+			render(root, a)
 			var spies = $window.__getSpies(a.dom)
 			a.dom.focus()
 
 			o(spies.valueSetter.callCount).equals(0)
 			o(a.dom.value).equals("1")
 
-			render(root, [b])
+			render(root, b)
 
 			o(spies.valueSetter.callCount).equals(0)
 			o(b.dom.value).equals("1")
 
-			render(root, [c])
+			render(root, c)
 
 			o(spies.valueSetter.callCount).equals(0)
 			o(c.dom.value).equals("1")
 
-			render(root, [d])
+			render(root, d)
 
 			o(spies.valueSetter.callCount).equals(1)
 			o(d.dom.value).equals("2")
 		})
 	})
-	o.spec("contenteditable attr throws on untrusted children", function() {
-		o("including text nodes", function() {
-			var div = {tag: "div", attrs: {contenteditable: true}, text: ""}
-			var succeeded = false
-
-			try {
-				render(root, div)
-
-				succeeded = true
-			}
-			catch(e){/* ignore */}
-
-			o(succeeded).equals(false)
-		})
-		o("including elements", function() {
-			var div = {tag: "div", attrs: {contenteditable: true}, children: [{tag: "script", attrs: {src: "http://evil.com"}}]}
-			var succeeded = false
-
-			try {
-				render(root, div)
-
-				succeeded = true
-			}
-			catch(e){/* ignore */}
-
-			o(succeeded).equals(false)
-		})
-		o("tolerating empty children", function() {
-			var div = {tag: "div", attrs: {contenteditable: true}, children: []}
-			var succeeded = false
-
-			try {
-				render(root, div)
-
-				succeeded = true
-			}
-			catch(e){/* ignore */}
-
-			o(succeeded).equals(true)
-		})
-		o("tolerating trusted content", function() {
-			var div = {tag: "div", attrs: {contenteditable: true}, children: [{tag: "<", children: "<a></a>"}]}
-			var succeeded = false
-
-			try {
-				render(root, div)
-
-				succeeded = true
-			}
-			catch(e){/* ignore */}
-
-			o(succeeded).equals(true)
-		})
-	})
-	o.spec("contentEditable prop throws on untrusted children", function() {
-		o("including text nodes", function() {
-			var div = {tag: "div", attrs: {contentEditable: true}, text: ""}
-			var succeeded = false
-
-			try {
-				render(root, div)
-
-				succeeded = true
-			}
-			catch(e){/* ignore */}
-
-			o(succeeded).equals(false)
-		})
+	o.spec("contenteditable throws on untrusted children", function() {
```

Comment:
The previous tests duplicated this sequence with a nebulous distinction between 'attr' & 'prop'